apptools/apptools-manifest-tests/README

```diff
@@ -0,0 +1,74 @@
+## Introduction
+
+This test suite is for wrt-manifest-android-tests
+
+## Precondition
+
+1. download and install allpairs
+   http://sourceforge.net/projects/allpairs/files/allpairs/
+2. Connect a Android device to your localhost
+3. Edit your input_seed.txt(in ./allpairs/) file to generate webapp.
+4. Need to edit the file "apptools-manifest-tests/arch.txt" according to the type of test device.
+   If test device is "arm" platform, content of the file should be "arm". If test device is "x86" platform, content of the file should be "x86".
+   If test devices are "arm" and "x86" platforms, content of the file should be "arm,x86", default value is "x86,arm"
+   If you want to test with 64 bit crosswalk, content of the file should be "arm,x86,64", default value is "x86,arm,32"
+5. Need to edit the file "apptools-manifest-tests/mode.txt" according to the type of build mode.
+   If create project with crosswalk lite zip, content of the file should be "lite". If create project with crosswalk zip and build "shared" mode apk, content of the file should be "shared". If create project with crosswalk zip and build "embedded" mode apk, content of the file should be "embedded".
+(Note: If you test "shared" mode, when uninstall crosswalk runtime library from android device before testing, it will be prompted to download crosswalk runtime library from google play in formal test process.)
+6. Need to edit the file "apptools-manifest-tests/platforms.txt" according to the type of test platform.
+   If test platform is Windows platform, content of the file should be "windows", it can also is "ios" or "deb", default value is "android".
+7. The node.js, Android SDK, JDK, apache ant, and git must be functional
+8. Install webp conversion tool from http://downloads.webmproject.org/releases/webp
+9. Download crosswalk-app-tools and release crosswalk zip, then install npm
+  9.1 clone source code from https://github.com/crosswalk-project/crosswalk-app-tools
+  9.2 Download the release Crosswalk zip which you want to test, and edit "../../VERSION"
+  9.3 Run commands: cd crosswalk-app-tools, then run:
+      npm install
+      npm install exec-sync
+10. Environment variable configuration:
+  10.1 Set DEVICE_ID env, DEVICE_ID can also be multiple ids like "DEVICE_ID=Medfield3C6DFF2E,Medfield3C6DFF00"
+  10.2 Set ANDORID_HOME env
+  10.3 The main script is crosswalk-app-tools/src/crosswalk-pkg . Set environment PATH or invoke with directory
+  10.4 Set CROSSWALK_APP_TOOLS_CACHE_DIR env to the release Crosswalk zip downloaded dir
+
+## Authors:
+
+* Yun, Liu<yunx.liu@intel.com>
+
+## LICENSE
+
+Copyright (c) 2016 Intel Corporation.
+Except as noted, this software is licensed under BSD-3-Clause License.
+Please see the COPYING file for the BSD-3-Clause License.
+
+1. allpairs is seed file folder:
```

Comment:
No title for these steps, and please move "authors" and “LICENSE” to the end of README.md